### ISSUE TITLE
Cannot find module 'jolocom-lib/js/resolver'

### ISSUE THREAD

**Issue by ChinmayPatel** [jolocom/jolocom-cli#19]
I'm trying to get started with this library and having issues with pretty much all the commands, including help. 

Steps to reproduce;

```
npm install -g jolocom-cli 
jolocom-cli -i fuel "32_keystring" // generated using crypto.randomBytes(32).toString('hex')
```

Stack trace; 

```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'jolocom-lib/js/resolver'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/chinmay/.nvm/versions/node/v10.22.0/lib/node_modules/jolocom-cli/dist/controller.js:24:20)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/chinmay/.nvm/versions/node/v10.22.0/lib/node_modules/jolocom-cli/dist/index.js:11:22)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/chinmay/.nvm/versions/node/v10.22.0/lib/node_modules/jolocom-cli/cli.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)

```

Does anyone already has a working fix?

**Comment by mnzaki** [comment]
This was a packaging issue, an older version of the lib was included. Fixed in the latest version, 1.9.2

**Comment by ChinmayPatel** [comment]
It works now! Having a `RegistryCommitFailed` but it's not related to this. Let me look first, and if I continue to get this error after a few changes, I'll report a new issues. 

Thanks for the prompt response.
